vespalog/src/main/java/com/yahoo/log/VespaLogHandler.java

```diff
@@ -73,12 +74,28 @@ public synchronized void publish(LogRecord record) {
         closeFileTarget();
     }
 
+    // Reduce log level from info level to fine level for some loggers
+    private Level possiblyReduceLogLevel(String loggerName, Level level) {
+        if (loggerName == null)
+            return level;
+
+        if (Set.of("com.yahoo.vespa.spifly.repackaged.spifly.BaseActivator",
```

Comment:
I guess this set should be in a "private static final" member variable, and the method should be static as well

Reply:
Agreed, done